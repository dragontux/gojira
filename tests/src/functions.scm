(display ((lambda () 'test)))
(newline)

(((lambda (x)
    (lambda (y)
      (x y)))
  (lambda (x)
    (display 'foo)
    (display x)
    (newline)))
 'thing)

(display ((lambda (foo)
            (* foo foo)) 123))
(newline)
