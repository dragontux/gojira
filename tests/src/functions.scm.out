'test
'foo'thing
15129
